cli_tools_e2e_test/gce_image_import_export/test_suites/import/cli_tests.go

```diff
@@ -32,11 +32,11 @@ import (
 )
 
 const (
-	testSuiteName = "ImageImportTests"
+	testSuiteName = "CLI"
```

Comment:
It's better to still have "import" keyword since export and import will be executed in one batch.

Reply:
sgtm - updated to ImageImportCLI